README.md

```diff
@@ -298,7 +298,6 @@ A collection of awesome things regarding React ecosystem.
 ##### React Projects
 * [gatsby - Transform plain text into dynamic blogs and websites using React.js](https://github.com/gatsbyjs/gatsby)
 * [react-blessed - A react renderer for blessed terminal interface library](https://github.com/Yomguithereal/react-blessed)
-* [MJML - The only framework that makes responsive-email easy. Based on React](https://mjml.io)
```

Comment:
Why is this removed?

Reply:
Not sure. I had a failing tests because some resources were not accessible. I see it's working now so I add it back.